### PR TITLE
Allow users to select a default channel, closes #373

### LINKED ISSUE #373: Add /g as default option
i would like to see the option to set the global chat as default, per player.

### PR DESCRIPTION
**Pull request by CaptainElliott**

### CODE REVIEW

**Comment by iEli2tyree011** [comment]
Your build failed

**Comment by CaptainElliott** [comment]
I don't belive it is my error, Cardinal was already failing from https://github.com/twizmwazin/CardinalPGM/commit/b6c3762e7e82579af8d70982628fa5e59fe45994
before I made this pr

**Comment by twizmwazin** [comment]
The build failures are because of Overcast's repo.  I'm in the process of getting my own repo. If anyone wants to help me out financially,  that would be cool.

**Comment by elespawol** [comment]
U can put a "Donation buttom".
El dia 11/03/2015 20.55, "Kevin Phoenix" <notifications@github.com> va
escriure:

**Comment by CaptainElliott** [comment]
This should build now, should I re-create the PR

**Comment by twizmwazin** [comment]
@CaptainElliott You don't need to, I re-ran the build

**Comment by CaptainElliott** [comment]
@iEli2tyree011 is player metadata not saved?

**Comment by iEli2tyree011** [comment]
@CaptainElliott Twiz and I both agree that they should be saved.

File: src/main/java/in/twizmwaz/cardinal/util/ChatUtils.java [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import in.twizmwaz.cardinal.module.modules.chatChannels.AdminChannel;
 import in.twizmwaz.cardinal.module.modules.chatChannels.GlobalChannel;
 import in.twizmwaz.cardinal.module.modules.chatChannels.TeamChannel;
+import jdk.nashorn.internal.objects.Global;
```
Comment (twizmwazin) [inline src/main/java/in/twizmwaz/cardinal/util/ChatUtils.java]: wrong import, causing build failure

**Comment by CaptainElliott** [comment]
Them being saved would take a lot more work, all I think only an external file can hold data after a restart

**Pull request merged**